srcpkgs/libgeotiff/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'libgeotiff'
+pkgname=libgeotiff
+version=1.7.1
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="proj-devel tiff-devel sqlite-devel"
+short_desc="TIFF based interchange format for georeferenced raster imagery"
+maintainer="Arjan Mossel <arjanmossel@gmail.com>"
+license="MIT"
+homepage="https://github.com/OSGeo/libgeotiff"
+distfiles="https://github.com/OSGeo/libgeotiff/releases/download/${version}/libgeotiff-${version}.tar.gz"
+checksum=05ab1347aaa471fc97347d8d4269ff0c00f30fa666d956baba37948ec87e55d6
+
+post_install() {
+	vlicense LICENSE
+	vmkdir usr/share/doc/libgeotiff
+	mv $DESTDIR/usr/doc/ $DESTDIR/usr/share/doc/libgeotiff/
+}
+
+libgeotiff-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/include
+		vmove "usr/lib/*.a"
```

Comment:
why only static?

Reply:
Here too there are no `.so` libs.

Reply:
Changed to use `BUILD_SHARED_LIBS=ON` (excludes static archives)